Custom.pm

```diff
@@ -11,24 +11,17 @@ package Slim::Utils::OS::Custom;
 
 use strict;
 use warnings;
-use Config;
-use File::Spec::Functions qw(:ALL);
-use FindBin qw($Bin);
-use base qw(Slim::Utils::OS);
 
-use constant MAX_LOGSIZE => 1024*1024*1; # maximum log size: 1 MB
+use base qw(Slim::Utils::OS::Linux);
 
-sub name {
-	return 'unix';
-}
+use constant MAX_LOGSIZE => 1024*1024*1; # maximum log size: 1 MB
+use constant UPDATE_DIR  => '/tmp/slimupdate';
```

Comment:
I missed that when I looked the first time.  If you've been running this, then I'm sure it's fine.